accessibility.html

```diff
@@ -0,0 +1,281 @@
+                    Placeholder text may not be read out, nor is it supported by older browsers.
+                  </li>
+                  <li>
+                    We can make sure the inputs are not autofocused, because a screenreader will lose all contextual information 
+                    if they are forced to jump to content in the middle of a page when they visit it.
+                  </li>
+                  <li>
+                    The button can be changed to be an actual semantic <em>&lt;button&gt;</em> rather than an <em>&lt;input&gt;</em> with its type set as button.
+                  </li>
+                  <li>
+                    Strongly coloured warning and success messages can be added to show the status of the form submission
+                  </li>
+                </ul>
+              </p>
+              <p>
+                <pre>
+                  <code class="language-html">
+                    &lt;form id="slack-integration"&gt;
+                      &lt;label for="mail-for-slack">Enter your email address&lt;/label&gt;
+                      &lt;input id="mail-for-slack" type="email" placeholder="me@mydomain.com" required>
+                      &lt;button type="submit" id="submit-slack">Get your invite&lt;/button&gt;
+                      &lt;div id="slack-form-message" class="message"&gt;&lt;/div&gt;
+                    &lt;/form&gt;
+                  </code>
+                </pre>
+              </p>
+              <img src="/src/img/accessibility-slack-after.png" alt="">
+              
+            </article> 
+          </section>
+          
+          <section class="cb">
+            <article id="tools" role="article">
+              <h2>Accessibility Testing</h2>
+              <p>
+                <ul>
+                  <li>
+                    Screenreaders like <a href="https://www.apple.com/uk/accessibility/osx/voiceover/">VoiceOver</a> 
+                    can help us to understand if our content makes sense and is structured well
+                  </li>
+                  <li>
+                    The NPM Package <a href="https://github.com/addyosmani/a11y">a11y</a> is great in pointing out accessibility issues, 
+                    as is the JavaScript library <a href="http://khan.github.io/tota11y/">tota11y</a>
+              
+                  </li>
+                  <li>
+                    The <a href="http://a11yproject.com/checklist.html">accessibility checklist</a>
+                    by the a11y project is a perfect place to make sure your components are accessible
+                  </li>
+                  <li>
+                    Writing tools like <a href="http://www.hemingwayapp.com/">Hemingway App</a>
+                    can help you simplify the words you write so everyone has a better chance of understanding them
+                  </li>
+                </ul>
```

Comment:
Needs a color contrast tool. I suggest http://wave.webaim.org/extension/, while not a perfect testing tool, the color piece is super easy and the extension lets you run it against localhost.